modules/passive-circuits.stanza

```diff
@@ -124,9 +124,9 @@ defn make-voltage-divider-module (in: JITXObject, out: JITXObject, lo: JITXObjec
     net (r-lo.p[1] r-hi.p[2] out)
     net (r-lo.p[2] lo)
 
-    val vin = typ(vo) * (property(r-lo.resistance) + property(r-hi.resistance)) / property(r-lo.resistance)
-    property(r-lo.operating-point) = OperatingPoint(Interval(0.0, typ(vo), false),       Interval(0.0, typ(vo) / property(r-lo.resistance), false))
-    property(r-hi.operating-point) = OperatingPoint(Interval(0.0, vin - typ(vo), false), Interval(0.0, (vin - typ(vo))/ property(r-hi.resistance), false))
+    val vin = typ-value(vo) * (property(r-lo.resistance) + property(r-hi.resistance)) / property(r-lo.resistance)
+    property(r-lo.operating-point) = OperatingPoint(Interval(0.0, typ-value(vo), false),       Interval(0.0, typ-value(vo) / property(r-lo.resistance), false))
```

Comment:
I don't think we're supporting interval anymore

Reply:
You are right -- that's for the next PR.